DNN Platform/DotNetNuke.Web.Client/ClientResourceSettings.cs

```diff
@@ -90,24 +90,34 @@ private UpgradeStatus Status
             }
         }
 
-        /// <summary>
-        ///
-        /// </summary>
-        /// <returns></returns>
+        [Obsolete("Deprecated in DotNetNuke 9.11.0.  Use overload taking portalId. Scheduled removal in v11.0.0.")]
         public bool IsOverridingDefaultSettingsEnabled()
         {
-            var portalVersion = GetIntegerSetting(PortalSettingsDictionaryKey, VersionKey);
-            var overrideDefaultSettings = GetBooleanSetting(PortalSettingsDictionaryKey, OverrideDefaultSettingsKey);
+            int? portalId = GetPortalIdThroughReflection();
+            return this.IsOverridingDefaultSettingsEnabled(portalId);
+        }
+
+        public bool IsOverridingDefaultSettingsEnabled(int? portalId)
+        {
+            var portalVersion = GetIntegerSetting(portalId, PortalSettingsDictionaryKey, VersionKey);
+            var overrideDefaultSettings = GetBooleanSetting(portalId, PortalSettingsDictionaryKey, OverrideDefaultSettingsKey);
 
             // if portal version is set
             // and the portal "override default settings" flag is set and set to true
             return portalVersion.HasValue && overrideDefaultSettings.HasValue && overrideDefaultSettings.Value;
         }
 
+        [Obsolete("Deprecated in DotNetNuke 9.11.0. Use overload taking portalId. Scheduled removal in v11.0.0.")]

```

Comment:
Changing version in deprecation message.
```suggestion
        [Obsolete("Deprecated in DotNetNuke 9.10.3.  Use overload taking portalId. Scheduled removal in v11.0.0.")]
```